### PR TITLE
adding in required dependecies

### PR DESCRIPTION
**Pull request by ssingh32**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,27 +4,32 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "date-fns": "latest",
     "@date-io/date-fns": "v1",
-    "@material-ui/core": "^4.9.10",
+    "@material-ui/core": "^4.9.11",
```
Comment (navjotsingh1996) [inline package.json:8]: Do you have to upgrade Material-UI?

File: package.json [hunk 2]
```diff
@@ -4,27 +4,32 @@
...
     "@material-ui/icons": "^4.9.1",
-    "@material-ui/pickers": "latest",
+    "@material-ui/pickers": "3.2.10",
```
Comment (navjotsingh1996) [inline package.json:10]: Can you validate that the date pickers still work? I had trouble when I first installed them and I had to keep specific versions for them. So can you just validate this is good (doesn't break anything)?

File: package.json [hunk 3]
```diff
@@ -4,27 +4,32 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "date-fns": "latest",
```
Comment (navjotsingh1996) [inline package.json:7]: Pretty sure we need this for the date picker

**Review by navjotsingh1996 — COMMENTED** [review]
Can you just validate that the date stuff works and there are no errors in the console? Ir not I think this is fine.

**Comment by ssingh32** [comment]
Screen Shot 2020-05-02 at 2 30 07 PM

get this error when trying to create a invoice

File: package.json (hunk 2 above)
Comment (ssingh32) [reply package.json:10]: yeah it still works

File: package.json (hunk 1 above)
Comment (ssingh32) [reply package.json:8]: not really but its just a bug version update so it should be fine

File: package.json (hunk 3 above)
Comment (ssingh32) [reply package.json:7]: It is still there check the package.json

**Pull request closed without merging**